db/migrate/20170815175201_swap_column_names.rb

```diff
@@ -0,0 +1,7 @@
+class SwapColumnNames < ActiveRecord::Migration
+  def change
+  	rename_column :scheduled_items, :admission, :admission_temp
+  	rename_column :scheduled_items, :audience, :admission
+  	rename_column :scheduled_items, :admission_temp, :audience
+  end
```

Comment:
Did you research this and test on staging to confirm that this is sound approach and won't lose data?